cmd/earlystopping/medianstop/v1beta1/Dockerfile

```diff
@@ -7,7 +7,10 @@ ENV PYTHONPATH ${TARGET_DIR}:${TARGET_DIR}/pkg/apis/manager/v1beta1/python
 
 RUN if [ "${TARGETARCH}" = "ppc64le" ] || [ "${TARGETARCH}" = "arm64" ]; then \
     apt-get -y update && \
-    apt-get -y install gfortran libopenblas-dev liblapack-dev && \
+    apt-get -y install gfortran libopenblas-dev liblapack-dev libssl-dev pkg-config gcc g++ && \
+    if [ "${TARGETARCH}" = "ppc64le" ]; then \
+      GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 pip3 install grpcio>=1.41.1; \
```

Comment:
why are we calling pip install here with specific version here? anyways we are calling the pip install below which does install the relevant version. Just that we need this `GRPC_PYTHON_BUILD_SYSTEM_OPENSSL` set to 1 only for the ppc64le platform.

Reply:
@mkumatag Its the version specified in requirements.txt. Hence I have added same version here. But it will install relevant version only.

Reply:
@mkumatag Updated dockerfile and have set flat `GRPC_PYTHON_BUILD_SYSTEM_OPENSSL` to 1 when arch is `ppc64le`